src/QueryM.purs

```diff
@@ -337,15 +338,12 @@ withMWalletAff
   :: forall (a :: Type). (Wallet -> Aff (Maybe a)) -> QueryM (Maybe a)
 withMWalletAff act = asks _.wallet >>= maybe (pure Nothing) (liftAff <<< act)
 
-callNami
+callWallet
```

Comment:
Maybe this should be `callCip30Wallet`, since it won't work with the testing wallet for example